Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/LoadTestDataWSServiceImpl.java

```diff
@@ -31,7 +31,6 @@
 
 import gov.hhs.fha.nhinc.admingui.constant.AdminWSConstants;
 import gov.hhs.fha.nhinc.admingui.services.LoadTestDataWSService;
-import gov.hhs.fha.nhinc.admingui.util.HelperUtil;
 import gov.hhs.fha.nhinc.common.loadtestdatamanagement.AddressType;
```

Comment:
Why LoadTestDataWSServiceImpl changes in PR?

Reply:
I was going to reused the GenericPortDescriptor<T> but decide to stick with StandardClass-PortDescripter after